### PR TITLE
Added debug info for journald

### LINKED ISSUE #612: todo: add info on how to debug journald and enable wrapping in journalctl less pager
Put info here: https://coreos.com/os/docs/latest/reading-the-system-log.html

set pager defaults `FRSXMK` to `FRXMK`

`SYSTEMD_LESS=FRXMK journalctl`

users always copy-paste what they see and sometimes copy-paste truncated logs.

info to read: http://www.freedesktop.org/software/systemd/man/journalctl.html#%24SYSTEMD_LESS

also add info on how to debug systemd-journald:

/etc/systemd/system/systemd-journald.service.d/10-debug.conf:
```
[Service]
Environment=SYSTEMD_LOG_LEVEL=debug
```

```
systemctl daemon-reload
systemctl restart systemd-journald
```

And retrieve debug messages:

```
dmesg | grep systemd-journald
```

### PR DESCRIPTION
**Pull request by kayrus**
/cc @robszumski 
resolves #612

### CODE REVIEW

File: os/reading-the-system-log.md [hunk 1]
```diff
@@ -63,5 +63,64 @@ journalctl -f
...
+
+```sh
+journalctl --no-pager
+```
+
+# Debugging journald
+
+If you've faced some problems with journald you can enable debug mode following the instructions below.
+
+## Enable debugging manually
```
Comment (robszumski) [inline os/reading-the-system-log.md]: title case

File: os/reading-the-system-log.md [hunk 2]
```diff
@@ -63,5 +63,64 @@ journalctl -f
 journalctl -u apache.service -f
 ```
 
+## Red Entries With Line Wrapping
+
+Read logs using default `less` pager with line wrapping:
```
Comment (robszumski) [inline os/reading-the-system-log.md]: Can we explain that we're just setting an env var before invoking the command?

And explain what the cryptic FRXMK value stands for?

**Comment by kayrus** [comment]
@robszumski fixed

File: os/reading-the-system-log.md [hunk 3]
```diff
@@ -63,5 +63,64 @@ journalctl -f
 journalctl -u apache.service -f
 ```
 
+## Red Entries With Line Wrapping
+
+By default `journalctl` passes `FRSXMK` command line options to [`less`](http://linux.die.net/man/1/less). You can override these options setting custom [`SYSTEMD_LESS`](http://www.freedesktop.org/software/systemd/man/journalctl.html#%24SYSTEMD_LESS) environment variable with omitted `S` option:
```
Comment (robszumski) [inline os/reading-the-system-log.md]: options **by** setting **a** custom

Comment (kayrus) [inline os/reading-the-system-log.md]: Fixed

**Comment by kayrus** [comment]
@robszumski @joshix I've fixed all links here.

File: os/network-config-with-networkd.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # Network Configuration with networkd
 
-CoreOS machines are preconfigured with [networking customized]({{site.baseurl}}/docs/sdk-distributors/distributors/notes-for-distributors) for each platform. You can write your own networkd units to replace or override the units created for each platform. This article covers a subset of networkd functionality. You can view the [full docs here](http://www.freedesktop.org/software/systemd/man/systemd-networkd.service.html).
+CoreOS machines are preconfigured with [networking customized](notes-for-distributors.md) for each platform. You can write your own networkd units to replace or override the units created for each platform. This article covers a subset of networkd functionality. You can view the [full docs here](http://www.freedesktop.org/software/systemd/man/systemd-networkd.service.html).
```
Comment (joshix) [inline os/network-config-with-networkd.md]: By default, CoreOS networking is configured for each deployment target or cloud provider. This document explains how to extend or replace those defaults by creating `networkd` units to describe your network.

This document covers only the relevant subset of `networkd` features. See the [full `networkd` documentation](http://www.freedesktop.org/software/systemd/man/systemd-networkd.service.html) for more.

**Comment by kayrus** [comment]
@joshix did you want to say something? I see your email, but can not see your comment here.

**Comment by joshix** [comment]
@kayrus I made a comment then changed my mind and deleted it.

File: os/reading-the-system-log.md [hunk 5]
```diff
@@ -63,5 +63,67 @@ journalctl -f
 journalctl -u apache.service -f
 ```
 
+## Red Entries With Line Wrapping
```
Comment (joshix) [inline os/reading-the-system-log.md]: "Read Entries with Line Wrapping" (typo and title case)

Comment (kayrus) [inline os/reading-the-system-log.md]: fixed

File: os/reading-the-system-log.md [hunk 6]
```diff
@@ -63,5 +63,67 @@ journalctl -f
 journalctl -u apache.service -f
 ```
 
+## Red Entries with Line Wrapping
```
Comment (joshix) [inline os/reading-the-system-log.md]: "Read" (unless I missed something and we are talking about the color Red)

**Pull request merged**